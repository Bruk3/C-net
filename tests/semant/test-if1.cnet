int main(int a, string argv) {
    int b = 4;
    if (a > b) {
        stdout.println("a is greater than b");
    }
    else
        stdout.println("a is less or equal to b");
    int c = 9;

}