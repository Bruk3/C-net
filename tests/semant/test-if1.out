int main(int a,string argv)
{
int b = (int : 4);
if ((int : (int : a) > (int : b)))
else{
(int : stdout.println((string : "a is less or equal to b")));
}
int c = (int : 9);
}

